src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/ColorRange.cs

```diff
@@ -62,7 +62,7 @@ private void UpdateColorRange()
             });
 
             // then update on the ui thread
-            t.ThenPost((_) =>
+            t.ThenSend((_) =>
```

Comment:
I found that if we didn't do this synchronously, the bitmap on the node was always one execution behind.